bin/tic_tac_toe.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './../lib/engine.rb'
+require_relative '../bin/user_interface.rb'
+# game running class
+class Game
```

Comment:
What is this class supposed to be doing? 

1. There isn't any initialize method which initializes all the data necessary for this class' instances
2. What sort of behavior (aka methods) should this class have? Does this class control the flow of the game? If not, what then?

Reply:
1. We don't initialize any method in the Game class because we already do that in the other class, we can just access it.

2. The game class just managing the call to the engine & user interface classes, so we can easily understand the code instead of cluttering the other classes with calls.

Reply:
Thanks Zeha for the reply. Kindly see my comments in the code snippets below.